ui/packages/catalog-ui-search/src/main/webapp/component/list-interactions/list-interactions.view.js

```diff
@@ -72,9 +74,36 @@ module.exports = Marionette.ItemView.extend({
     const newList = new this.model.constructor(copyAttributes)
     this.model.collection.add(newList)
   },
-  triggerAction(event) {
-    const url = event.currentTarget.getAttribute('data-url')
-    window.open(url)
+  async triggerAction(event) {
+    let url = event.currentTarget.getAttribute('data-url')
+
+    let res = await fetch(url, { Accept: 'application/json' })
+
+    try {
+      if (!res.ok) {
+        throw Error(res.statusText)
+      }
+
+      if (res.headers.get('content-disposition') != null) {
+        this.downloadAsFile(res)
```

Comment:
Could we reuse https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/react-component/utils/save-file/save-file.tsx#L14?